lib/profile/index.js

```diff
@@ -98,6 +100,8 @@ function displayedDefaults (final_result) {
     profile.offline_hotspot = allDefaults.offline_hotspot;
     profile.edison_battery_shutdown_voltage = allDefaults.edison_battery_shutdown_voltage;
     profile.pi_battery_shutdown_percent = allDefaults.pi_battery_shutdown_percent;
+    profile.threshold_setting = allDefaults.profile.threshold_setting;
+    profile.smb_delivery_ratio = allDefaults.smb_delivery_ratio;
```

Comment:
These don't necessarily need to be displayedDefaults for rig users. I believe adding them to defaults above would be sufficient to allow iAPS to display them (or not) as y'all see fit?

Does anyone have an opinion on whether they should also be displayedDefaults for OpenAPS rig users? We generally have trie to avoid putting too many customizations "up front" for people to feel they should tweak them. Do the defaults here seem good enough for most users that these can be buried deeper to change if people really need to? Or do we need to make them more visible to encourage people to experiment with / tweak them?